File: admin-emahu/read_lint.js
const fs = require('fs');
const path = 'lint_output.txt';

if (fs.existsSync(path)) {
  const content = fs.readFileSync(path, 'utf16le');
  console.log("--- Full ESLint Error Lines ---");
  const lines = content.split('\n');
  for (const line of lines) {
    if (line.includes('1019') || line.includes('1028') || line.includes('1038') || line.includes('3309')) {
      console.log(line.trim());
    }
  }
} else {
  console.log("File not found");
}

File: admin-emahu/read_lint.py
import os

path = r"e:\emahu\Website---Emahu\admin-emahu\lint_output.txt"
if os.path.exists(path):
    # Try different encodings
    for encoding in ['utf-16', 'utf-16-le', 'utf-16-be', 'utf-8', 'latin-1']:
        try:
            with open(path, 'r', encoding=encoding) as f:
                content = f.read()
                if len(content.strip()) > 0:
                    print(f"--- Encoding: {encoding} ---")
                    # print lines that contain "error" or "warning"
                    lines = content.split('\n')
                    for line in lines:
                        if 'error' in line.lower() or 'warning' in line.lower() or 'problems' in line.lower():
                            print(line)
                    break
        except Exception as e:
            pass
else:
    print("File not found")
